comparison.tex

```diff
@@ -0,0 +1,36 @@
+\section{Comparison}
+
+We present a few alternatives for performing proof of burn, all of which are captured by our burn protocol definition.
+
+\noindent
+\textbf{OP\_RETURN.}
```

Comment:
For these we need the citation to the guy from the bitcoin wiki...

Reply:
which guy? If you mean Iain Stewart, these methods don't appear in his work